challenges/logic/todo/luka_jelacic/lib/blocs/todo_state.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:luka_jelacic/todoItem.dart';
+
+enum TodoStateStatus {
+  InitialState,
+  NewItemState,
+  DeleteItemState,
+  DuplicateItemState,
+  EmptyItemState
+}
+
+class TodoState {
+  TodoStateStatus todoStatus;
+  List<TodoItem> todos;
+  TodoState({this.todoStatus, this.todos});
```

Comment:
TodoState({this.todoStatus, this.todos**,**});

Zarez na kraju liste parametara ukoliko ima vise od 1